protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -369,6 +411,17 @@ func (k Keeper) UpdateSubaccounts(
 			),
 		)
 
+		// if GRPC streaming is on, emit a generated subaccount update to stream.
+		if streamingManager := k.GetFullNodeStreamingManager(); streamingManager.Enabled() {
```

Comment:
Nit for later: we might want to early return if fns doesn't have a current subscription to this subaccount id. This is definitely a hot path in our code.

Reply:
done

Reply:
I think we can't have that outside the FNS manager due to accessing this without the lock. Think we need to define:

def streamingManager.CheckIfSubaccountRegistered :
    lock.acquire()
    defer lock.release
    // check subaccount to subscirption id mapping

Reply:
done, created `TracksSubaccountId` function instead.